### DISCUSSION TITLE
nobody cares that no way to send money out from wallet ?

### DISCUSSION THREAD

**Discussion by srrsr** [Chia-Network/chia-blockchain#6670] [Support]
No way to enter amount . Always error that no numeric value put in . Win version bug .

**Comment by loppefaaret** [comment]
you have to put a number into both the amount to send, and the amount of fee, let me suggest `0` fee - never `1` - that would send your transaction through with 1 XCH fee

**Comment by BrandtH22** [comment]
@srrsr , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
